docs/supported-container-runtimes.md

```diff
@@ -10,6 +10,20 @@ Works out of the box.
 export DOCKER_HOST=unix://${XDG_RUNTIME_DIR}/podman/podman.sock
 ```
 
+### Known issues
+
+**Resource reaper does not work on MacOS**
+
+The resource reaper is a process which runs in the background and cleans up containers and images after they are no longer needed. It is not possible to run the resource reaper on MacOS with Podman due to permission limitations, it fails with "Operation not supported" errors.
```

Comment:
Should we mention the rootful workaround?

Reply:
I still had network related issues with the rootful workaround. Is it not the case in .NET?

Reply:
I've just double checked, I do not see network issues, perhaps yesterday I confused something with the ryuk privileged flag. Disabling ryuk works for both rootful and rootless